tensorflow/compiler/xla/service/gpu/matmul_utils.cc

```diff
@@ -377,7 +378,11 @@ StatusOr<se::blas::ComputationType> GetBlasComputationType(
       return se::blas::ComputationType::kF32;
     case F32:  // fall-through
     case C64:
-      return se::blas::ComputationType::kTF32AsF32;
+      if (tsl::tensor_float_32_execution_enabled()) {
+        return se::blas::ComputationType::kTF32AsF32;
```

Comment:
It seems this also affects the cuBlas non-LT path as well. This function is called by DoGemmWithAlgorithm, which is used in the non-LT codepath.

https://github.com/tensorflow/tensorflow/blob/bcc062239f0f2c10e84b1dc4b04ff4ff3aa27ecc/tensorflow/compiler/xla/service/gpu/matmul_utils.cc#L450-L462

cuBlasLT is currently off by default, so do these tests fail on Ampere currently without cuBLAS LT? Anyway, this doesn't require any code changes, but if this affects cuBlas non-LT, update the PR description.

Reply:
Updated PR description.